dolphinscheduler-python/pydolphinscheduler/examples/yaml_define/Dependent_External.yaml

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Define the process
+process:
+  name: "task_dependent_external"
+
+# Define the tasks under the process
+tasks:
+  - {
+      "task_type": "Shell",
+      "params": { "name": "task_1", "command": "echo task 1" },
+    }
```

Comment:
I think this yaml format is a little odd, can we find some better for the yaml like https://github.com/argoproj/argo-workflows/blob/master/examples/dag-daemon-task.yaml

Reply:
@caishunfeng  do you have time to take a look the yaml template, I know you have learn other workflow engine using yaml file